tests/tasks/post.yml

```diff
@@ -1,19 +1,6 @@
 # post test file
 ---
-- name: install test dependencies
-  ansible.builtin.apt:
-    name:
-      - wget
-    state: "{{ apt_install_state | default('latest') }}"
-
-- name: test rstudio-workbench installation
-  ansible.builtin.shell: >
-    wget http://localhost:8787/auth-sign-in -O /dev/null -S --quiet 2>&1
-    | grep -q '200 OK'
-    && (echo 'Availability test: pass' && exit 0)
-    || (echo 'Availability test: fail' && exit 1)
-  args:
-    warn: false
-  changed_when: false
-  tags:
-    - skip_ansible_lint
+- name: Test rstudio-workbench installation
+  ansible.builtin.uri:
+    url: http://localhost:8787/auth-sign-in
+    status_code: 200
```

Comment:
Health check using Ansible module (instead of doing `wget` and bash stuff which cased error: `risky-shell-pipe Shells that use pipes should set the pipefail option.`).

Reply:
You forgot the tags. However, I'm not sure if it's used.

Reply:
`skip_ansible_lint` is not used